src/details/ArborX_DetailsTags.hpp

```diff
@@ -28,7 +28,7 @@ struct BoxTag
 {
 };
 
-template <typename Geometry>
+template <typename T, typename Enable = void>
```

Comment:
Wouldn't it make sense to separate geometry tags and predicate tags? Is there any function that is supposed to handle both?

Reply:
And put them what header instead?  This is good enough.